lib/vanagon/driver.rb

```diff
@@ -182,6 +182,17 @@ def render # rubocop:disable Metrics/AbcSize
       @project.make_makefile(workdir)
     end
 
+    def dependencies(temp_dir) # rubocop:disable Metrics/AbcSize
+      # Simple sanity check for the project
+      if @project.version.nil? or @project.version.empty?
```

Comment:
`or` does surprising things in Ruby (I've read appeals that it, along with `and` be deprecated). Always favor `||`